proposals/NNNN-allow-let-property-wrapper.md

```diff
@@ -0,0 +1,239 @@
+# Allow Property Wrappers on Let Declarations
+
+* Proposal: [SE-NNNN](NNNN-allow-let-property-wrapper.md)
+* Authors: [Amritpan Kaur](https://github.com/amritpan), [Pavel Yaskevich](https://github.com/xedin)
+* Review Manager: [Xiaodi Wu](https://github.com/xwu)
+* Status: **Awaiting review**
+* Implementation: [apple/allow-let-property-wrapper](https://github.com/apple/swift/pull/62342) 
+
+## Introduction
+
+[SE-0258 Property Wrappers](https://github.com/apple/swift-evolution/blob/main/proposals/0258-property-wrappers.md) models both mutable and immutable variants of a property wrapper but does not allow wrapped properties explicitly marked as a `let`. This proposal builds on the original vision by expanding property wrapper usage to include let declared properties with a wrapper attribute.
+
+## Motivation
+
+Allowing property wrappers to be applied to `let` properties improves Swift language consistency and expressivity.
+
+Today, a property wrapper can be applied to a property of any type but the rules for declaring these wrapped properties are varied and singular to property wrappers. For example, a struct instance without the property wrapper attribute can be declared with either a `var` or a `let` property. However, mark the struct as a property wrapper type and the compiler no longer allows it to be written as a `let` wrapped property:
```

Comment:
```suggestion
Today, a property wrapper cannot be applied to a property declared with `let`:

```

Edits for clarity and brevity. There is no need to tell users that properties can be declared with `var` or `let`; it isn't necessary to specify that the containing type is a `struct`; and the language is ambiguous as to which of the two structures in the example is "the" struct being described—nor is it necessary.

Extra newline for compatibility with a wider range of Markdown renderers.